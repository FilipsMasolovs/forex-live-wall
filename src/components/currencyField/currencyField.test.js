import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import CurrencyField from './currencyField';

const currencies = [{ticker: "EUR/USD"},{ticker: "USD/JPY"},{ticker: 'GBP/USD'}]

it('renders without crashing', () => {
  const parent = document.createElement('div')
  ReactDOM.render(<CurrencyField currencies={currencies} />, parent)
})

it('renders correctly', () => {
  const tree = renderer
    .create(<CurrencyField currencies={currencies} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});