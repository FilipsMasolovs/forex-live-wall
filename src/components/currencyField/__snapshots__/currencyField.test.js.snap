// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<select
  className="forex-live-wall--currency-select"
  onChange={[Function]}
  required={true}
>
  <option
    disabled={true}
    value=""
  >
    Choose a Currency Pair
  </option>
  <option
    value="EUR/USD"
  >
    EUR/USD
  </option>
  <option
    value="USD/JPY"
  >
    USD/JPY
  </option>
  <option
    value="GBP/USD"
  >
    GBP/USD
  </option>
</select>
`;
